src/main/java/chess/domain/game/ChessGame.java

```diff
@@ -3,20 +3,26 @@
 import chess.domain.board.Board;
 import chess.domain.board.coordinate.Coordinate;
 import chess.domain.game.state.BlackTurn;
+import chess.domain.game.state.End;
 import chess.domain.game.state.Start;
 import chess.domain.game.state.State;
 import chess.domain.game.state.WhiteTurn;
 import chess.domain.piece.Team;
 
 public class ChessGame {
 
+    public static final String END_STATE = "END";
     private State state;
 
     public ChessGame() {
         state = new Start();
     }
 
     public ChessGame(Board board, String state) {
+        if (state.equals(END_STATE)) {
```

Comment:
```suggestion
        if (END_STATE.equals(state)) {
```

NPE 방지를 위해 더 확실한 값을 앞에 두고 비교하는 습관을 들이는 것이 좋습니다 :)